.github/workflows/ci.yaml

```diff
@@ -53,4 +53,6 @@ jobs:
       with:
         path: wayfire-plugins-extra
     - run: cd wayfire-plugins-extra && wget https://raw.githubusercontent.com/WayfireWM/wayfire/master/uncrustify.ini
-    - run: cd wayfire-plugins-extra && git ls-files | grep "hpp$\|cpp$" | xargs ../uncrustify/build/uncrustify -c uncrustify.ini --check
+    - run: cd wayfire-plugins-extra && git ls-files | grep "hpp$\|cpp$" | xargs ../uncrustify/build/uncrustify -c uncrustify.ini --no-backup --replace
+    - run: cd wayfire-plugins-extra && git diff
+    - run: cd wayfire-plugins-extra && git diff | diff - /dev/null &> /dev/null
```

Comment:
If I understand correctly, you are using this as a way to make the CI fail by passing it the return value of diff? Instead of these two lines, what about using `git diff --cached --exit-code`?

Reply:
Is there a reason for `--cached`? The diff should  not be staged.